Description: Use FHS compliant path for data
 mercurial installs expects its data files next to their py files in
 /usr/lib/python2.7/dist-packages/mercurial .  We put data file in the
 fhs compliant /usr/share/mercurial .
Author: Javi Merino <vicho@debian.org>

--- a/mercurial/util.py
+++ b/mercurial/util.py
@@ -578,8 +578,10 @@ def mainfrozen():
 if mainfrozen():
     # executable version (py2exe) doesn't support __file__
     datapath = os.path.dirname(sys.executable)
-else:
+elif sys.argv[0] == "gendoc.py":
     datapath = os.path.dirname(__file__)
+else:
+    datapath = "/usr/share/mercurial"
 
 i18n.setdatapath(datapath)
 
